### PR TITLE
Tutorial 1 file - Instructions on how to address a 'requirements not met' or 'pip unable to connect' error

### PR DESCRIPTION
**Pull request by BaanCraft**
I inserted notifications of what to do if an error were to be encountered during the tutorial. It contains a set of instructions on what to do if the message "Unable to install requirements. This may be because one of your requirements is invalid, or because pip was unable to connect to the PyPI server" was encountered

This solves an issue where the virtual environment is set up with an unsupported version of python (ie 3.13), leading to the resulting error.

## PR Checklist:

- [x] All new features have been tested
- [x] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by BaanCraft** [comment]
Apologies, I was suffering from a condition known as "I did not know what I was doing and this was my first pull request", leading to the errors.

File: docs/tutorial/tutorial-1.rst [hunk 1]
```diff
@@ -219,6 +219,33 @@ This should open a GUI window:
        :align: center
        :alt: Hello World Tutorial 1 window, on Windows
 
+    .. admonition:: Invalid requirements or pip unable to connect
+
+      You may encounter an error that reads:
```
Comment (freakboy3742) [inline docs/tutorial/tutorial-1.rst]:
```suggestion
      If you encounter the error:
```

File: docs/tutorial/tutorial-1.rst [hunk 2]
```diff
@@ -219,6 +219,23 @@ This should open a GUI window:
...
+
+      If you encounter the error:
+
+      .. code-block:: doscon
+
+        Unable to install requirements. This may be because one of your
+        requirements is invalid, or because pip was unable to connect
+        to the PyPI server.
+
+
```
Comment (freakboy3742) [inline docs/tutorial/tutorial-1.rst:232]:
```suggestion

```

**Review by freakboy3742 — APPROVED** [review]
One minor whitespace change, but otherwise, this is good to land! Thanks for the contribution!

File: docs/tutorial/tutorial-1.rst [hunk 3]
```diff
@@ -219,6 +219,21 @@ This should open a GUI window:
...
+    .. admonition:: Invalid requirements or pip unable to connect
+
+      If you encounter the error:
+
+      .. code-block:: doscon
+
+        Unable to install requirements. This may be because one of your
+        requirements is invalid, or because pip was unable to connect
+        to the PyPI server.
+      Confirm that you are running a :ref:`version of python that this tutorial supports
```
Comment (freakboy3742) [inline docs/tutorial/tutorial-1.rst:231]:
```suggestion
        to the PyPI server.

      Confirm that you are running a :ref:`version of python that this tutorial supports
```

**Pull request merged**